mat64/format_example_test.go

```diff
@@ -13,18 +13,19 @@ import (
 func ExampleFormatted() {
 	a := mat64.NewDense(3, 3, []float64{1, 2, 3, 0, 4, 5, 0, 0, 6})
 
-	// Create a matrix formatting value with a prefix ...
-	fa := mat64.Formatted(a, mat64.Prefix("    "))
+	// Create a matrix formatting value with a prefix and calculating each column width individually...
```

Comment:
Please split this over two lines (think about the formating in godoc - or run `godoc -http :6060` locally and check).